skore/src/skore/ui/project_routes.py

```diff
@@ -69,6 +70,11 @@ def __serialize_project(project: Project) -> SerializedProject:
         elif isinstance(item, MediaItem):
             value = base64.b64encode(item.media_bytes).decode()
             media_type = item.media_type
+        elif isinstance(item, CrossValidationItem):
+            # Convert plot to MediaItem
+            item = MediaItem.factory(item.plot)
+            value = base64.b64encode(item.media_bytes).decode()
+            media_type = item.media_type
```

Comment:
Take a look at my comment on `cross_validate_item.py`.

Does the user really need access to the plot in the backend? 
- If yes (what i see in `cross_validate.py` in a jupyter context), you should add a property `plot` which can dynamically reconstruct the plot from the persisted plot (json).
- If not, you can here just pass the persisted plot (json) as bytes.

Reply:
done